python/ray/dataframe/utils.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pandas as pd
+import ray
+
+
+def _get_lengths(df):
+    """Gets the length of the dataframe.
+
+    Args:
+        df: A remote pd.DataFrame object.
+
+    Returns:
+        Returns an integer length of the dataframe object. If the attempt
+            fails, returns 0 as the length.
+    """
+    try:
+        return len(df)
+    # Because we sometimes have cases where we have summary statistics in our
+    # DataFrames
+    except TypeError:
+        return 0
+
+
+def from_pandas(df, npartitions=None, chunksize=None):
+    """Converts a pandas DataFrame to a Ray DataFrame.
+
+    Args:
+        df (pandas.DataFrame): The pandas DataFrame to convert.
+        npartitions (int): The number of partitions to split the DataFrame
+            into. Has priority over chunksize.
+        chunksize (int): The number of rows to put in each partition.
+
+    Returns:
+        A new Ray DataFrame object.
+    """
+    from .dataframe import DataFrame
```

Comment:
Why is the import here? (not suggesting to do it differently, just curious)

Reply:
Had an issue with relative imports, but if there's a better alternative I would be happy to replace this

Reply:
It's a cyclical import because we import `utils.py` from `dataframe.py`.